sources/packages/web/src/views/student/financial-aid-application/StudentApplication.vue

```diff
@@ -19,16 +19,20 @@
       </v-btn>
     </template>
   </ModalDialogBase>
-  <v-sheet elevation="1" class="mx-auto">
-    <v-container>
-      <formio
-        formName="studentapplication"
-        :data="initialData"
-        @loaded="formLoaded"
-        @customEvent="customEventCallback"
-      ></formio>
-    </v-container>
-  </v-sheet>
+  <student-page-container>
+    <template #content>
+      <v-sheet elevation="1" class="mx-auto">
```

Comment:
we can use v-card instead of v-sheet

Reply:
I didn't change any existing code to keep it simple.. checked the UI if it looked like before 😊